environments/liveness/package.json

```
@@ -1,8 +1,8 @@
 {
   "private": true,
-  "name": "liveness-environment",
+  "name": "environments-in-app-messaging",
```

I know it's draft still but reminder to self to fix name here 😀

---

```suggestion
  "name": "environments-liveness",
```
Let's do it now 😈

---

Let's do it now, this is me not copy pasting properly 😈 

```suggestion
  "name": "environments-liveness",
```